src/Sentry.Unity.Editor/Native/BuildPostProcess.cs

```diff
@@ -40,43 +41,83 @@ public static void OnPostProcessBuild(BuildTarget target, string pathToProject)
+        {
+            var crashpadPath = Path.GetFullPath(Path.Combine("Packages", SentryPackageInfo.GetName(), "Plugins",
+                "Windows", "Sentry", "crashpad_handler.exe"));
+            var targetPath = Path.Combine(projectDir, Path.GetFileName(crashpadPath));
+            logger.LogInfo($"Copying the native crash handler '{Path.GetFileName(crashpadPath)}' to the output directory");
+            File.Copy(crashpadPath, targetPath, true);
+        }
+
+        private static void uploadDebugSymbols(IDiagnosticLogger logger, string projectDir, string executableName)
+        {
+            var cliOptions = SentryCliOptions.LoadCliOptions();
+            if (!cliOptions.Validate(logger))
+                return;
+
+            var propertiesFile = SentryCli.CreateSentryProperties(projectDir, cliOptions);
+
+            logger.LogInfo($"Uploading debuging information using sentry-cli in {projectDir}");
+
+            var paths = "";
+            Func<string, bool> addPath = (string name) =>
+            {
+                var fullPath = Path.Combine(projectDir, name);
+                if (Directory.Exists(fullPath) || File.Exists(fullPath))
+                {
+                    paths += " " + name;
+                    logger.LogDebug($"Adding '{name}' to the debug-info upload");
+                    return true;
+                }
+                else
+                {
+                    logger.LogWarning($"Coudn't find '{name}' - debug symbol upload will be incomplete");
+                    return false;
+                }
+            };
+
+            addPath(executableName);
+            addPath("GameAssembly.dll");
+            addPath("UnityPlayer.dll");
+            addPath(Path.GetFileNameWithoutExtension(executableName) + "_BackUpThisFolder_ButDontShipItWithYourGame");
+
+            // no debug info in sentry.dll but sentry.io will complain about a missing file...
+            // addPath(Path.GetFileNameWithoutExtension(executableName) + "_Data/Plugins/x86_64/sentry.dll");
+
+            // Configure the process using the StartInfo properties.
+            var process = new Process
+            {
+                StartInfo = new ProcessStartInfo
+                {
+                    FileName = SentryCli.SetupSentryCli(),
+                    WorkingDirectory = projectDir,
+                    Arguments = "upload-dif " + paths,
+                    UseShellExecute = false,
+                    RedirectStandardOutput = true,
+                    RedirectStandardError = true,
+                    CreateNoWindow = true
+                }
+            };
+            process.StartInfo.EnvironmentVariables["SENTRY_PROPERTIES"] = propertiesFile;
+            process.OutputDataReceived += (sender, args) => logger.LogDebug($"sentry-cli: {args.Data.ToString()}");
+            process.ErrorDataReceived += (sender, args) => logger.LogWarning($"sentry-cli: {args.Data.ToString()}");
```

Comment:
Why not level error?

```suggestion
            process.ErrorDataReceived += (sender, args) => logger.LogError($"sentry-cli: {args.Data.ToString()}");
```